services/userService.js

```diff
@@ -0,0 +1,39 @@
+const { set, get } = require('./cacheService')
+const {
+  fetchUser
+} = require('../models/users')
+
+const cacheUser = (user) => {
+  if (!user) {
+    return false
+  }
+  const userObject = {
+    github_id: user.github_id,
+    id: user.id,
+    username: user.username
+  }
+  set(user.id, userObject)
```

Comment:
What's the TTL here?
According to the docs, this will stay in cache forever. We need to expose the cache set, get wrapper functions from the cache service which will have proper defaults and then use it here. 

Also, how will we manage the invalidation/updation of the data in the cache after user caches his profile info?